src/transmission/scala/mrtjp/projectred/transmission/wireabstracts.scala

```diff
@@ -243,9 +243,11 @@ abstract class WirePart(wireType:WireType) extends TMultiPart with TWireCommons
 
     override def getStrength(player:PlayerEntity, hit:PartRayTraceResult) = 2/30f
 
-    override def getOutlineShape = new IndexedVoxelShape(WireBoxes.sShapes(getThickness)(side), 0)
+    override def getInteractionShape: VoxelShape = getShape(ISelectionContext.empty())
```

Comment:
This line can be removed, because it does the same thing as the overridden method

Reply:
Good point; didn't catch that!